src/css/_variables.scss

```diff
@@ -4,7 +4,6 @@ $secondary-text: #F4A460; // currently just used for visited links in errors and
 $primary-bg: #000;
 $secondary-bg: lighten($primary-bg, 35%);
 
-$icon-font-family: 'VideoJS';
```

Comment:
If someone was overriding it with there own, they'd probably want to override all of the icons partial basically (at least that was my thought).